notebooks/inference_SavedModel_VertexEndpoint.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<p>Add some references to the official docs from which we borrowed code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/semantic-segmentation-ml-pipeline/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>